canvas_course_admin_tools/settings/local.py

```diff
@@ -42,6 +42,9 @@
     'icommons_rest_api': {
         'base_path': 'api/course/v2'
     },
+    'manage_course': {
+        'relative_url': 'courses/27/external_tools/170',  # dev (Manage Course)
```

Comment:
Yup, that's the plan!  (And I remembered it from your last PR feedback for the canvas_account_admintools_tool).  :)   I haven't done it yet because the Manage People is currently down for me (note scrumteam chat), so I'm not able to run any of the existing manage people tests to make sure they all run as they should. The plan is to remove the manage people URL once I can successfully run them since the dashboard click should take care of the need to explicitly provide the tool URL anymore.   (On second thought, I could probably do this ahead of time in this PR...will fix shortly - thanks!)